netbox/netbox_inventory.py

```diff
@@ -132,53 +182,84 @@ def fetch_inventory(cli_opts):
     if cli_opts.region:
         params["region"] = cli_opts.region
 
-    res = netbox.get("/api/dcim/devices/", params=params)
+    res = NETBOX_SESSION.get("/api/dcim/devices/", params=params)
```

Comment:
**Suggestion:** Add error handling for the `requests.get` method to manage exceptions like `requests.exceptions.RequestException` which can occur during network issues or invalid responses. [error handling, importance: 8]
```suggestion
    try:
        res = NETBOX_SESSION.get("/api/dcim/devices/", params=params)
        res.raise_for_status()
    except requests.exceptions.RequestException as e:
        sys.exit(f"Failed to fetch devices: {str(e)}")
```